src/systems/ec/implementations/react/templates/pages/research-area-hub/src/ResearchAreaHubPage.jsx

```diff
@@ -0,0 +1,1301 @@
+                }}
+              />
+
+              <SocialMediaFollow
+                className="ecl-u-mt-l"
+                description="Follow the latest progress and learn more about getting involved."
+                links={[
+                  {
+                    href: '/example',
+                    label: 'Twitter',
+                    variant: 'standalone',
+                    iconPosition: 'before',
+                    icon: [
+                      {
+                        shape: 'twitter',
+                        size: 'xl',
+                      },
+                      {
+                        shape: 'twitter_hover',
+                        size: 'xl',
+                      },
+                    ],
+                  },
+                  {
+                    href: '/example',
+                    label: 'Facebook',
+                    variant: 'standalone',
+                    iconPosition: 'before',
+                    icon: [
+                      {
+                        shape: 'facebook',
+                        size: 'xl',
+                      },
+                      {
+                        shape: 'facebook_hover',
+                        size: 'xl',
+                      },
+                    ],
+                  },
+                  {
+                    href: '/example',
+                    label: 'Instagram',
+                    variant: 'standalone',
+                    iconPosition: 'before',
+                    icon: [
+                      {
+                        shape: 'instagram',
+                        size: 'xl',
+                      },
+                      {
+                        shape: 'instagram_hover',
+                        size: 'xl',
+                      },
+                    ],
+                  },
+                  {
+                    href: '/example',
+                    label: 'LinkedinIn',
+                    variant: 'standalone',
+                    iconPosition: 'before',
+                    icon: [
+                      {
+                        shape: 'linkedin',
+                        size: 'xl',
+                      },
+                      {
+                        shape: 'linkedin_hover',
+                        size: 'xl',
+                      },
+                    ],
+                  },
+                  {
+                    href: '/example',
+                    label: 'Other social networks',
+                    variant: 'standalone',
+                  },
+                ]}
+              />
+
+              <h2
+                className="ecl-u-type-heading-2 ecl-u-mb-l ecl-u-mt-lg-3xl ecl-u-type-color-black"
+                id="events"
+              >
+                Related events
+              </h2>
+
+              <article className="ecl-u-border-bottom ecl-u-border-color-grey-15 ecl-u-d-flex ecl-u-pv-m">
+                <div className="ecl-u-flex-grow-0 ecl-u-mr-m">
```

Comment:
right margin should be xl `ecl-u-mr-m`

Reply:
Not updated, but I put this as resolved as other CT have the same right margin on date_block.